schema/config-schema.json

```diff
@@ -147,6 +147,32 @@
                 "noNewPrivileges": {
                     "type": "boolean"
                 },
+                "scheduler": {
+                    "type": "object",
+                    "properties": {
+                        "policy": {
+                            "type": "string"
+                        },
```

Comment:
This should probably have an enum as well (`SCHED_OTHER`, `SCHED_FIFO` ...), and perhaps a definition (to match the Go type)